multi-vterm.el

```diff
@@ -80,15 +80,15 @@ If nil, this defaults to the SHELL environment variable."
     (switch-to-buffer vterm-buffer)))
 
 ;;;###autoload
-(defun multi-vterm-projectile ()
+(defun multi-vterm-project ()
   "Create new vterm buffer."
   (interactive)
-  (if (projectile-project-p)
-      (if (buffer-live-p (get-buffer (multi-vterm-projectile-get-buffer-name)))
-          (if (string-equal (buffer-name (current-buffer)) (multi-vterm-projectile-get-buffer-name))
+  (if (project-current)
```

Comment:
It works fine for the git (vc) project. But, I don't know to mark a directory ~/Dropbox/org-modes at a project without git. When I used `projectile`, I just need to create an empty file `.projectile` to mark a normal directory as a project.

Reply:
Well, you can do something like this to have project.el treat dirs with `.project` as project roots:

```elisp
(defun my/project-find-dot-project (dir)
  (when-let ((root (locate-dominating-file dir ".project")))
    `(dot-project . ,root)))

(cl-defmethod project-roots ((project (head dot-project)))
  (list (cdr project)))

(push #'my/project-find-dot-project project-find-functions)
```

This piece of code is inspired by similar codes from others. Hope it can meet your needs.

Reply:
It works! Thank you.